integration/sql/iface-java/src/java/io/openlineage/sql/OpenLineageSql.java

```diff
@@ -0,0 +1,15 @@
+package io.openlineage.sql;
+
+import java.util.List;
+
+public final class OpenLineageSql {
+    public static native SqlMeta parse(List<String> sql, String dialect, String defaultSchema) throws RuntimeException;
+    public static SqlMeta parse(List<String> sql, String dialect) throws RuntimeException { return parse(sql, dialect, null); }
+    public static SqlMeta parse(List<String> sql) throws RuntimeException { return parse(sql, null, null); }
+    
+    public static native String provider();
+    
+    static {
+        System.loadLibrary("openlineage_sql_java");
```

Comment:
I have a very basic script that is importing the library but I'm receiving the following error works:

```text
Exception in thread "main" java.lang.UnsatisfiedLinkError: no openlineage_sql_java in java.library.path
        at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1860)
        at java.lang.Runtime.loadLibrary0(Runtime.java:871)
        at java.lang.System.loadLibrary(System.java:1124)
        at io.openlineage.sql.OpenLineageSql.<clinit>(OpenLineageSql.java:13)
        at App.main(App.java:15)
```

I'm compiling the below script with `javac -cp ".:../path/to/integration/sql/iface-java/target/openlineage-sql-java-1.0-SNAPSHOT.jar" App.java` and executing it similarly with `java -cp ".:../path/to/integration/sql/iface-java/target/openlineage-sql-java-1.0-SNAPSHOT.jar" App.java`. 

I can see the so file in `/integration/sql/target/debug/` but I'm not sure how to include it in the class path.  Shouldn't it get bundled with the jar file itself?

```java
import io.openlineage.sql.OpenLineageSql;
import io.openlineage.sql.SqlMeta;

import java.util.ArrayList;
import java.util.Arrays;

public class App {
    public String getGreeting() {
        return "Hello World!";
    }

    public static void main(String[] args){
        System.out.println(new App().getGreeting());
        SqlMeta output = OpenLineageSql.parse(Arrays.asList("SELECT * FROM test"));
        System.out.println(output.toString());
    }
}
```

Reply:
Looking into it

Reply:
The new commit should fix that